packages/fast-components-react-msft/src/select-option/examples.data.tsx

```diff
@@ -47,5 +47,8 @@ export default {
                 },
             } as any,
         },
+        {
+            children: "html children",
```

Comment:
this seems confusing, wouldn't it just show the string "html children"? in which case its just a string child.

Reply:
I wanted to simply show that children passed to the component would render, you mean I should have some other element here like a div?

Reply:
I suppose it would be less confusing if it said something like "Hello world" or "Foo" because I'm trying to take some kind of meaning away from "html children" and it just reads as a confusing string.

Reply:
"Hello world" it is.